SFDigitalServices/sfgov
# Sg 0000 - campaign

- Create a vocabulary Campaign themes
- Create a hook update to prepopulate Campaign theme terms
- Create and theme Campaign content type and its paragraphs
- Theming Campaign 

**How to test:** 
- Create a new node of type Campaign
- Verify that the node looks good in both mobile and desktop
